src/app/templates/app/submit_new_license.html

```diff
@@ -352,12 +352,14 @@
               data: form,
               success: function (data) {
                 var githubCode = data.statusCode;
+                var githubIssueId = data.issueId;
                 if(githubCode == '201'){
                   var successMessage = "The license request has been successfully submitted.";
                   $("#messages").html('<div class="alert alert-success alert-dismissable fade in"><a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a><strong>Success! </strong>'+ successMessage +'</div>');
-                      setTimeout(function() {
-                        $("#messages").html("");
-                      }, 7000);
+                  setTimeout(function() {
+                    $("#messages").html("");
+                  }, 7000);
+                  window.open("https://github.com/spdx/licnse-list-xml/issues/" + githubIssueId, "_blank");
```

Comment:
nit: Let's have this as constant - `https://github.com/spdx/licnse-list-xml/issues/`